packages/api/src/graphql/mutation.private.ts

```diff
@@ -344,20 +353,20 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       args: {
         email: {type: GraphQLNonNull(GraphQLString)}
       },
-      async resolve(
-        root,
-        {url, email},
-        {authenticate, dbAdapter, generateJWT, mailContext, urlAdapter}
-      ) {
+      async resolve(root, {email}, {authenticate, prisma, generateJWT, mailContext, urlAdapter}) {
         const {roles} = authenticate()
         authorise(CanSendJWTLogin, roles)
 
-        const user = await dbAdapter.user.getUser(email)
+        const user = await prisma.user.findUnique({
+          where: {email: email}
```

Comment:
`where: {email}`

Reply:
Should probably have this as a prettier/eslint setting.